cookbook/security/remember_me.rst

```diff
@@ -16,17 +16,29 @@ the session lasts using a cookie with the ``remember_me`` firewall option:
 
         # app/config/security.yml
         firewalls:
-            main:
+            default:
+                # ...
                 remember_me:
                     key:      "%secret%"
                     lifetime: 604800 # 1 week in seconds
                     path:     /
+                    # by default, the feature is enabled using
+                    # a checkbox in the login form (see below),
+                    # uncomment the below lines to always enable
+                    # it.
```

Comment:
I find this lonely `it.` word a bit disturbing. Could we move it upwards, please?